packages/spicetify-creator/src/scripts.ts

```diff
@@ -10,7 +10,7 @@ const autoprefixer = require("autoprefixer");
 
 const exec = promisify(require('child_process').exec);
 
-const build = async (watch: boolean, minify: boolean, outDirectory?: string) => {
+const build = async (watch: boolean, minify: boolean, outDirectory?: string, inDirectory?: string) => {
   const settings: ICustomAppSettings & IExtensionSettings = JSON.parse(fs.readFileSync("./src/settings.json", 'utf-8'));
```

Comment:
I think this also needs to be using `inDirectory`
Also, what do you think of doing a
```ts
if (!inDirectory) inDirectory = './src';
```
at the start of the function instead of the duplicate `inDirectory || './src'`?

Reply:
done